crew

```diff
@@ -1037,7 +989,7 @@ def install_command (args)
   args["<name>"].each do |name|
     @pkgName = name
     search @pkgName
-    print_current_package
+    print_current_package true
```

Comment:
`print_current_package @opt_verbose`?

Reply:
The `true` option is used to show the version and homepage

Reply:
I know but don't you think it should be optional?

Reply:
I think the `version` should show in first install (just like crew 1.5.5), maybe making a new option called `@info` ?

Reply:
I'm not necessarily in agreement.  What do you think @satmandu?

Reply:
I don't think that showing version and homepage during a package install makes it too chatty. But I am biased because when I do installs during package testing it is really useful to have that info, without having the lest of the verbose info being dumped onto the screen.

Reply:
So does that mean you are not in favor of keeping it optional?

Reply:
Since there are differing opinions on what level of verbosity is warranted, should  opt_verbose not be a binary option but instead a level?